site/es/blog/new-in-devtools-96/index.md

```diff
@@ -28,171 +28,172 @@ draft: true
 
 {% include 'partials/devtools/es/banner.md' %}
 
-<!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
+## Función de vista previa: Nuevo panel de Descripción General de CSS {: #css-overview }
 
-<!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
-[Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
+Utilice el nuevo panel **Descripción General de CSS** para identificar posibles mejoras de CSS en su página.
```

Comment:
It seems that we have different translations for CSS Overview in  "Spanish" and in "Spanish (Latin America)". This is unfortunate and should be fixed along with other inconsistencies with the newly added translations. However, in the mean time, could we please include the other translation ("Resumen de CSS") here as well?
```suggestion
Utilice el nuevo panel **Descripción General de CSS / Resumen de CSS** para identificar posibles mejoras de CSS en su página.
```